server/domain/student/models/Student.ts

```
@@ -0,0 +1,161 @@
+// server/domain/student/models/Student.ts
+import { Id } from '../../shared/models/Id';
+import { AcademicInfo, StudentStatus, StudentStatusType } from './AcademicInfo';
```

ESLint

[error] Imports "StudentStatusType" are only used as type.

(@typescript-eslint/consistent-type-imports)